### PR TITLE
test: update access token

### PR DESCRIPTION
**Pull request by n-riesco**
* Regenerated access key and token.

* Moved oauth2 tests into its own file.

* Refactored common functions into `test/backend/utils.js`.

* Added script to run oauth2 tests only.

### CODE REVIEW

**Comment by n-riesco** [comment]
@tarzzz would you review this one?

File: test/backend/routes.spec.js [hunk 1]
```diff
@@ -122,292 +51,6 @@ function closeServers() {
 
 // Suppressing ESLint cause Mocha ensures `this` is bound in test functions
 /* eslint-disable no-invalid-this */
-describe('Servers:', () => {
```
Comment (tarzzz) [inline test/backend/routes.spec.js]: I see you have removed these tests, but I cant find where you added them..

File: test/backend/routes.oauth2.spec.js [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+        // Save some connections to the user's disk
+        saveSetting('USERS', [{
+            username, apiKey
+        }]);
+        saveSetting('SSL_ENABLED', false);
+
+        // ensure fetch starts with no cookies
+        clearCookies();
+    });
```
[the remark this answers is not in the document]
Comment (n-riesco) [reply test/backend/routes.oauth2.spec.js]: Done! (now we have `createTestServers` and `closeTestServers` in `utils.js`)

File: test/backend/routes.spec.js (hunk 1 above)
Comment (n-riesco) [reply test/backend/routes.spec.js]: Excellent catch!

**Comment by n-riesco** [comment]
@tarzzz Ready for review again.

**Comment by tarzzz** [comment]
💃 .. Thanks for doing those changes.

**Pull request merged**